src/Common/src/Interop/Windows/BCrypt/AesBCryptModes.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Security.Cryptography;
+using Internal.NativeCrypto;
+
+namespace Internal.Cryptography
+{
+    internal static class AesBCryptModes
+    {
+        private static readonly SafeAlgorithmHandle s_hAlgCbc = OpenAesAlgorithm(Cng.BCRYPT_CHAIN_MODE_CBC);
+        private static readonly SafeAlgorithmHandle s_hAlgEcb = OpenAesAlgorithm(Cng.BCRYPT_CHAIN_MODE_ECB);
+
+        internal static SafeAlgorithmHandle GetSharedHandle(CipherMode cipherMode)
+        {
+            // Windows 8 added support to set the CipherMode value on a key,
+            // but Windows 7 requires that it be set on the algorithm before key creation.
```

Comment:
I'm probably just being dense, but how does this comment apply to the following code?